src/sql/platform/query/common/queryRunner.ts

```diff
@@ -347,11 +347,10 @@ export default class QueryRunner extends Disposable {
 			let hasShowPlan = !!result.resultSetSummary.columnInfo.find(e => e.columnName === 'Microsoft SQL Server 2005 XML Showplan');
 			if (hasShowPlan) {
 				this._isQueryPlan = true;
-				this.getQueryRows(0, 1, result.resultSetSummary.batchId, result.resultSetSummary.id).then(e => {
-					if (e.resultSubset.rows) {
-						this._planXml.resolve(e.resultSubset.rows[0][0].displayValue);
-					}
-				});
+				const e = await this.getQueryRows(0, 1, result.resultSetSummary.batchId, result.resultSetSummary.id);
```

Comment:
Don't think we want to await here, this code can safely be run out of context of the rest of this code.

Reply:
Will sync offline